<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>desaSimple</class>
 <widget class="QWidget" name="desaSimple">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modificar</string>
  </property>
  <widget class="QRadioButton" name="r_IN">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Configurar com a Entrada</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="r_OUT">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Configurar com a Sortida</string>
   </property>
  </widget>
  <widget class="QPushButton" name="desa">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Desa la Configuracio</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
